Source/Charts/Renderers/LineChartRenderer.swift

```diff
@@ -171,17 +191,7 @@ open class LineChartRenderer: LineRadarRenderer
             drawCubicFill(context: context, dataSet: dataSet, spline: fillPath!, matrix: valueToPixelMatrix, bounds: _xBounds)
         }
 
-        if dataSet.isDrawLineWithGradientEnabled
-        {
-            drawGradientLine(context: context, dataSet: dataSet, spline: cubicPath, matrix: valueToPixelMatrix)
-        }
-        else
-        {
-            context.beginPath()
-            context.addPath(cubicPath)
-            context.setStrokeColor(drawingColor.cgColor)
-            context.strokePath()
-        }
+        drawLine(context: context, dataSet: dataSet, spline: cubicPath, matrix: valueToPixelMatrix, drawingColor: drawingColor)
```

Comment:
I thought about doing this. I think this correct way to do it is
```
if dataSet.isDrawLineWithGradientEnabled
{
    drawGradientLine(context: context, dataSet: dataSet, spline: cubicPath, matrix: valueToPixelMatrix)
}
else
{
    drawLine(in: ...)
}
```

Reply:
I exposed this login into a separate function with the name `drawLine`.

Reply:
I'm aware. I'm saying that `drawLine` and `drawGradientLine` should be separate methods.

Reply:
Just pushed a change. I think that this's correct now.